sdk/include/platform/sunburst/platform-pwm.hh

```diff
@@ -0,0 +1,66 @@
+#pragma once
+#include <debug.hh>
+#include <stdint.h>
+
+/**
+ * A driver for Sonata's PWM.
```

Comment:
Understood - I intend to maintain the `Pwm` in the struct name so that we only have to use `SonataPwm` instead of `SonataPulseWidthModulation` every time we want to instantiate PWM - be sure to let me know if you want this changed as well. Instead, I've updated the high-level documentation to expand the abbreviation so that it is clear what it is referring to, and have mostly scrubbed references to the abbreviation from other parts of the code.

Reply:
No, please use `SonataPulseWidthModulation`.  If someone wants to use a shorter local name, they can use a `using` directive.

Reply:
Understood, I'll make that change now.